chapters/dashboard/connect-jinaD.md

```diff
@@ -0,0 +1,48 @@
+#  Connect Dashboard to JinaD
+
+Before using the Dashboard with Jina, you'll need to connect it to JinaD first. <br/>
+Jinad is a REST + Websockets based server to allow remote workflows in Jina. <br/>
```

Comment:
add link to the JinaD repo

Reply:
Hasn't JinaD been integrated into the Core?